cmd/agent/app/reporter/flags.go

```diff
@@ -28,8 +28,8 @@ import (
 const (
 	// Whether to use grpc or tchannel reporter.
 	reporterType = "reporter.type"
-	// Agent tags
-	agentTagsDeprecated = "jaeger.tags"
+	// AgentTagsDeprecated is a configuration property name for adding process tags to incoming spans.
+	AgentTagsDeprecated = "jaeger.tags"
```

Comment:
Should this be named `JaegerTagsDeprecated` as not specific to agent?
Assuming not specific to agent - the changes seem limited to the agent config at the moment - shouldn't they also be applied to collector?

Reply:
Collector uses only `--collector.tags`